core/validatorapi/validatorapi.go

```diff
@@ -217,6 +218,11 @@ func (c *Component) RegisterAwaitBeaconBlock(fn func(ctx context.Context, slot i
 	c.awaitBlockFunc = fn
 }
 
+// RegisterTracker registers the tracker component.
+func (c *Component) RegisterTracker(fn func(context.Context, core.Duty, core.ParSignedDataSet)) {
```

Comment:
no need for this, can use existing `RegisterParSigDB` (should probably rename to `Subscribe`)